configuration/rules-dsl.md

```diff
@@ -101,8 +101,9 @@ then
 end
 ```
 
-A rule can have any number of trigger conditions, but must at least have one.
-The `SCRIPT_BLOCK` contains the code that should be executed, when a trigger condition is met, see the [script](#scripts) section for details on its syntax.
+- Each rule must have a unique name within your openHAB installation, and the name should be wrapped in quotes.
+- A rule can have any number of trigger conditions, but must at least have one.
+- The `SCRIPT_BLOCK` contains the code that should be executed, when a trigger condition is met, see the [script](#scripts) section for details on its syntax.
```

Comment:
I was thinking about something similar to:

- `<RULE_NAME>` - Each rule must have a unique name (given in quotes). It is recommended to choose a speaking meaningful description as rule name.
- `<TRIGGER_CONDITION>` - A rule is executed in reaction to at least one or multiple trigger conditions. Please see below for different possible triggers.
- `<SCRIPT_BLOCK>` - Contains the logic that should be executed when a trigger condition is met. See the [script](#scripts) section for details on its code syntax.